go.mod

```diff
@@ -36,26 +36,26 @@ replace go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.open
 replace github.com/jackc/pgx/v4 => github.com/jackc/pgx/v5 v5.5.4
 
 // These replace directives support the backlevel go version required by ODH build
-replace github.com/project-codeflare/appwrapper v0.23.0 => github.com/project-codeflare/appwrapper v0.23.1-0.20240731154950-ad486fb7e7ee
+replace github.com/project-codeflare/appwrapper v0.24.0 => github.com/project-codeflare/appwrapper v0.24.1-0.20240828152845-bfaac68eef30
 
-replace sigs.k8s.io/kueue v0.7.1 => github.com/opendatahub-io/kueue v0.7.0-odh-test
+replace sigs.k8s.io/kueue v0.8.0 => github.com/opendatahub-io/kueue v0.7.0-odh-test
```

Comment:
This is a bit of a hack... but the PR to upgrade the ODH fork to Kueue 0.8 isn't merged yet and the bits of the Kueue go API that AppWrapper depends on were not changed from 0.7 to 0.8. 

We can either hold this PR until https://github.com/opendatahub-io/kueue/pull/44 is merged and refer to the new tag here _or_ we can merge this now and then update the replace once the ODH PR merges.  I believe both work.

Reply:
I'm fine with merging this for now, and following up with an update when the sync PR in ODH gets merged.

Reply:
Sounds good.  

Now that you are in the owners file, I think you may need to /approve again to get the bot to add the label and proceed with the merge.